certifikit-cli/src/main/kotlin/app/cash/certifikit/cli/Main.kt

```diff
@@ -132,27 +141,35 @@ class Main : Callable<Int> {
     }
   }
 
-  private fun completeOption() {
+  private suspend fun completeOption() {
     if (complete == "host") {
       for (host in knownHosts()) {
         println(host)
       }
     }
   }
 
-  internal fun addHostToCompletionFile(host: String) {
+  @Suppress("BlockingMethodInNonBlockingContext")
+  internal suspend fun addHostToCompletionFile(host: String) {
     val previousHosts = knownHosts()
     val newHosts = previousHosts + host
 
     val lineSeparator = System.getProperty("line.separator")
-    knownHostsFile.writeText(newHosts.joinToString(lineSeparator, postfix = lineSeparator))
+    filesystem.sink(knownHostsFile.also { filesystem.createDirectories(it.parent!!) }).buffer().use {
```

Comment:
the lack of a createDirectories call on the old code is surprising!

Reply:
Any suggestions?